loop.py

```diff
@@ -59,6 +60,41 @@ def compute_unfairness(A, R):
     return np.abs(np.asarray(A) - np.asarray(R))
 
 
+def compute_max_unfairness(A, R, k):
+    min_value = sys.maxsize
+    unfairness = (np.asarray(A) - np.asarray(R))[::-1]
+    index = len(unfairness) - 1
+    result = -1
+
+    for item in unfairness:
+        if index > k-1 and item < min_value:
```

Comment:
is never going to be true

Reply:
You are right, I got it the wrong way around after I got it right at first. Will change asap.

Reply:
Are you sure? The variable index starts as the max index of the list and decreases as time goes on and so can and will be larger than k-1. After becoming smaller than k-1, this will be false, which is desired behavior as it makes no sense to swap a random top-k item with another top-k item.

Reply:
item < sys.max_value